simple.css

```diff
@@ -649,6 +649,18 @@ dialog::backdrop {
   }
 }
 
+/* Superscript & Subscript */
+/* Prevent scripts from affecting line-height. */
+sup, sub {
+  vertical-align: baseline;
+  position: relative;
+  top: -0.4em;
```

Comment:
Shouldn't this rule be in a separate ruleset dedicated to the `sup` selector? It feels redundant to define it here for `sub` too, and then override it immediately after with the next ruleset. It would also make the code easier to read IMO. WDYT @djpeacher @kevquirk?

Reply:
Great suggestion! Seems obvious in retrospect 😅